### DISCUSSION TITLE
I have a question and need your help.

### DISCUSSION THREAD

**Discussion by EasyRuby** [Monika-After-Story/MonikaModDev#7777] [QA - tech support + general]
My MAS no longer displays notifications，it is really a problem for me,when monika want to say something to me,I can't receive her information in time.I hope everyone can help me, thank you very much.
![1629530992002](https://user-images.githubusercontent.com/64263797/130314383-9645b581-fe35-4650-9c23-2ae30fd86e1b.jpg)
![W28GHZ`MT36PYQUFAYA~)K5](https://user-images.githubusercontent.com/64263797/130314384-a33049af-d935-4c21-8d2e-a3c7aa1e41c5.png)

**Comment by KritRom** [comment]
You have to go to the Alerts tab and check Use Notifications and Window Reacts. Let me know if the problem persists.
